<?xml version="1.0"?>
<launch>
    <node name="teleopduck2" pkg="teleop_twist_keyboard" type="teleop_twist_keyboard.py" output="screen">
    <remap from="/cmd_vel" to="/cmd_vel/duck2"/>
    </node>
</launch>
